src/main/scala/com/trueconnectivity/e2e/utils/HealthCheck.scala

```diff
@@ -0,0 +1,41 @@
+package com.trueconnectivity.e2e.utils
+
+import scalaj.http.{Http, HttpResponse}
+
+import scala.util.{Failure, Success, Try}
+
+object HealthCheck {
+  def waitUntilHealthy(applicationPort: Int, retryCount: Int, retryDelay: Int): Unit = {
+    retry(retryCount, retryDelay) {
+      Http(s"http://localhost:${applicationPort}/health/status")
```

Comment:
just thinking if we want tho have the host configurable, but probably in 99,999 % noc necessary, if somebody would require another they could update the library

Reply:
The only time i have had to do this is when devs have Windows machines that can't run Docker for Windows, and when you had to run Boot2Docker on Macs